backend/decky_loader/utilities.py

```diff
@@ -195,7 +195,7 @@ async def http_request(self, req: Request) -> StreamResponse:
                 self.logger.debug(f"Finished stream for {url}")
         return res
 
-    async def http_request_legacy(self, method: str, url: str, extra_opts: Any = {}):
+    async def http_request_legacy(self, method: str, url: str, extra_opts: Any = {}, timeout: int | None = None):
```

Comment:
Unused argument?

Reply:
intended to prevent kwargs error, no point actually implementing since its only for legacy compatibility but i might look into it

Reply:
implemented properly now